modules/account-lib/src/coin/xtz/transactionBuilder.ts

```diff
@@ -68,7 +71,7 @@ export class TransactionBuilder extends BaseTransactionBuilder {
   /**
    * Public constructor.
    *
-   * @param _coinConfig
+   * @param _coinConfig {CoinConfig }- coin configuration
```

Comment:
Usually type param goes before the name, and I think it's mostly unnecessary in typescript projects

Reply:
Sounds good, was coerced into adding this by the linter :) can change the placement though